packages/react-scripts/template/src/registerServiceWorker.js

```diff
@@ -74,7 +74,7 @@ function registerValidSW(swUrl) {
 
 function checkValidServiceWorker(swUrl) {
   // Check if the service worker can be found. If it can't reload the page.
-  fetch(swUrl)
+  window.fetch(swUrl)
```

Comment:
Hmm. I don't think we want to do this. Pretty much all tutorials use `fetch` without qualifier and I don't want to confuse readers.

I'm fine with other changes but I'd rather keep this one as is. If somebody uses Standard they can feel free to change it by hand in their project.

Reply:
a `standard` user also has the option of adding 

```json
"standard": { "globals": "fetch" }
```

to their package.json